src/styles/styles.js

```diff
@@ -982,6 +982,7 @@ const styles = {
 
     statusIndicator: {
         borderColor: themeColors.sidebar,
+        backgroundColor: themeColors.buttonDangerBG,
```

Comment:
Question: How come the background color change here? I feel like this also affects other locations this is used no?

Reply:
No, it only has one usage